packages/libs/react-ui/src/components/Card/Card.css.ts

```diff
@@ -0,0 +1,43 @@
+import { sprinkles, vars } from '../../styles';
+
+import { style } from '@vanilla-extract/css';
+
+export const container = style([
+  sprinkles({
+    background: 'neutral2',
+    color: 'neutral6',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    paddingX: 'lg',
+    paddingY: 'md',
+    borderRadius: 'sm',
+    marginY: 'md',
+    border: 'none',
+  }),
+  {
+    width: 'max-content',
```

Comment:
could you add this to the sprinkles now that you've added it to the sprinkles file?